app/frontend/styles/application.scss

```diff
@@ -91,3 +91,20 @@ p {
     display: none;
   }
 }
+
+.no-border-override {
+  border: none !important;
+}
+
+a.govuk-tabs__tab.tab-links-override:link,
+.govuk-tabs__tab.tab-links-override:visited,
+a.js-enabled .govuk-tabs__tab.tab-links-override:link,
+a.js-enabled .govuk-tabs__tab.tab-links-override:visited {
+    color: $govuk-link-colour;
+}
+
+li.govuk-tabs__list-item--selected > a.govuk-tabs__tab.govuk-tabs__tab.tab-links-override, 
```

Comment:
Could we separate these concerns into the most specific file e.g. `app/styles/components/vacancies`?

Reply:
Yep, will get that done